idx/memory/tag_query.go

```diff
@@ -368,9 +367,19 @@ func NewTagQuery(expressions []string, from int64) (TagQuery, error) {
 // getInitialByEqual generates the initial resultset by executing the given equal expression
 func (q *TagQuery) getInitialByEqual(expr kv, idCh chan schema.MKey, stopCh chan struct{}) {
 	defer q.wg.Done()
+	key, err := idx.IdxIntern.GetPtrFromByte([]byte(expr.key))
```

Comment:
this seems to be a common repeating pattern that `.key` and `.value` need to be retrieved from interning, and if there's an error it must be logged and `internError` needs to be increased. wouldn't it be easier to just add a method for that to `kv`?

Reply:
Yes and no, each case warrants different behavior so it is difficult to add just one method